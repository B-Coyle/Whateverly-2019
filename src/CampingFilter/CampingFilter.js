import React, {Component} from 'react';
import './CampingFilter.scss';
import CampCard from '../CampCard/CampCard.js';

export default class Camping extends Component {
  constructor(props) {
    super(props);

  }

  render () {
    let Card = this.props.filteredCampgrounds.map((campground) => {
      return (<CampCard 
        filteredCampgrounds = {campground}
        key = {campground.name}
        allTrails = {this.props.allTrails}
        assignedUserCampInput = {this.props.userCampInputValue}
      />)

    })
    
    return (
      <section className='campCardContainer'>{ Card }</section>
    )
  }
}
