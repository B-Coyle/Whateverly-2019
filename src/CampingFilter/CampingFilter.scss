.campCardContainer {
  margin: 5px;
  display: block;
  grid-column-start: 2;
  grid-row-start: 2;
}

