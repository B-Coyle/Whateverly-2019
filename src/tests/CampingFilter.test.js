import React from 'react';
import ReactDom from 'react-dom';
import CampingFilter from '../CampingFilter/CampingFilter.js';
import { shallow } from 'enzyme';

const mockCamping = [
    {
    "name": "Glacier Basin Campground",
    "location": "Estes Park",
    "season": "May-Sept",
    "elevation": 8500,
    "camping": ["Tent", "Car", "RV"],
    "activities": ["Hiking", "Fishing", "Horseback Riding", "Wildlife"],
    "reservation-available": true,
    "nearest-city": null
      },
      {
    "name": "Grand Junction KOA",
    "location": "Grand Junction",
    "season": "All year",
    "elevation": 4600,
    "camping": ["Tent", "Car", "RV"],
    "activities": ["ATVing"],
    "reservation-available": true,
    "nearest-city": null
    }
]

describe('CampingFilter', () => {
let wrapper

beforeEach(() => {
    wrapper = shallow(
        <CampingFilter filteredCampgrounds={mockCamping}/>
    )
});

// it('should mock the snapshot', () => {
//     expect(wrapper).toMatchSnapshot();
// });

it('should have a default state', () => {
    expect(wrapper.state()).toEqual({
    })
});

})