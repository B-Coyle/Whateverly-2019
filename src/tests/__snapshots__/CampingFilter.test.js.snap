// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampingFilter should mock the snapshot 1`] = `undefined`;
